precipitation/ngsild-payloads/Ouvrage.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "urn:ngsi-ld:Structure:OPR0000144308",
+    "type": "Structure",
+    "stationCode": {
```

Comment:
you're missing this fix: https://github.com/easy-global-market/ngsild-api-data-models/pull/62/commits/9afa127f82c87a193db368c2bcd890f24d3ce7ad